README.md

```diff
@@ -171,6 +173,10 @@ $ rm -rf tanka
    
 **Solution:** This is likely because the jaeger agent is not running correctly. Check that all pods were successfully scheduled.
 
+**Issue:** ./install.sh fails complaining about Ingress. You're using a `k3d` that runs a too new version of k8s. That means that Ingress got promoted and the `extensions.v1beta1.ingress` in [production/sample/default/main.jsonnet] is not valid anymore.
+
+**Solution:** You are likely using a version of `k3d` that is too new. Verify it running kubectl version. As per [this blogpost](https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/) k8s v1.22 and higher no longer server that API.
```

Comment:
```suggestion
**Solution:** You are likely using a version of `k3d` that is new. Verify the kubectl version. As per this [blogpost](https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/) k8s v1.22 and higher no longer serve that API.
```

Reply:
Please feel free to re-read the phrasing and let me know what you think @achatterjee-grafana